app/src/main/java/com/keylesspalace/tusky/db/dao/TimelineStatusDao.kt

```diff
@@ -183,6 +183,16 @@ WHERE tuskyAccountId = :tuskyAccountId AND serverId = :statusId"""
     )
     abstract suspend fun setVoted(tuskyAccountId: Long, statusId: String, poll: String)
```

Comment:
I just found out that one can do this:
```suggestion
    @TypeConverters(Converters::class)
    abstract suspend fun setVoted(tuskyAccountId: Long, statusId: String, poll: Poll)
```
and then there is no more need for the manual json conversion with Moshi

Reply:
🤩